### ISSUE TITLE
Can't change settings on Firefox

### ISSUE THREAD

**Issue by Gvanderl** [jamestomasino/stutter#97]
Whenever I change any setting (wpm, shortcuts etc), the changes don't save and get reset to default. This happens through the settings page and through shortcuts (alt+up arrow). I've tried disabling all other extensions and it still doesn't work. I've tried on the same machine on chrome and it works there.
Hereby attached is the console log of the error when I try to change wpm.

<img width="753" alt="Screen Shot 2022-05-25 at 14 42 25" src="https://user-images.githubusercontent.com/27513709/170265081-e18643e1-2aa0-48a4-9ed0-ca241b94daf0.png">

**Comment by jamestomasino** [comment]
What version of Firefox are you using?

**Comment by Gvanderl** [comment]
I'm on 91.9.0esr (64-bit), on MacOS Monterey on a M1 pro MBP. 
Looking into it, it might be the software required by my workplace that runs on my machine to keep it safe. No idea why it would interfere with your extension but if that's the issue then I'm afraid there isn't much you can do about it.

**Comment by jamestomasino** [comment]
That's the issue after all. Object.hasOwn was released to replace Object.hasOwnProperty, but it didn't get into Firefox until version 92. Knowing there's ESR versions older than that out there I'm going to need to put in some fallbacks. Thanks for the heads up.

**Comment by Gvanderl** [comment]
Indeed, updating Firefox to 100 fixed the issue. Keeping this open until it's fixed for ESR.
